priv/catalog/373DB8.yaml

```diff
@@ -6,7 +6,9 @@ description: |
 remediation: |
   ## Abstract
   The fencing timeout (`stonith-timeout`) determines the time Pacemaker will wait for fencing to succeed.
-  The recommended values on Azure are `144` seconds for SBD only or `900` seconds when using SBD combined with the Azure Fence agent.
+  The remommended default value is at least `150` seconds for SBD.
+  The recommended values on Azure are at least `144` seconds for SBD only or `900` seconds when using SBD combined with the Azure Fence agent.
```

Comment:
@angelabriel I would super quickly rewrite this to be like:
```
The recommended values on Azure...
The recommended value on AWS..
The recommended value on GCP...
```

Otherwise it looks like AWS and GCP are kind of coming together and look like less important than Azure

Reply:
Thanks for the suggestion. We don't want to hurt any of our partners.